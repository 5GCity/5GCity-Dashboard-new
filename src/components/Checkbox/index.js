/**
 * Checkbox Component
 * Please write a description
 *
 * @author Your Name <youremail@ubiwhere.com>
 */
import React from 'react'
import styled from 'styled-components'
import { Checkbox } from 'element-react'

export default ({ children, ...props }) => (
  <Checkbox {...props}>Checkbox</Checkbox>
)

const Wrapper = styled.div``
