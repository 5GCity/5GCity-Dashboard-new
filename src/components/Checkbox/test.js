/**
 * Checkbox Component Tests
 *
 * @author Your Name <youremail@ubiwhere.com>
 */
import React from 'react'
import renderer from 'react-test-renderer'
import Checkbox from './index'
import 'jest-styled-components'

it('renders correctly', () => {
  const tree = renderer
    .create(<Checkbox />)
    .toJSON()

  expect(tree).toMatchSnapshot()
})
