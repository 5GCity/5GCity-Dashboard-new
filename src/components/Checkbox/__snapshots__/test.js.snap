// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<label
  className="el-checkbox"
  style={Object {}}
>
  <span
    className="el-checkbox__input"
  >
    <span
      className="el-checkbox__inner"
    />
    <input
      checked={false}
      className="el-checkbox__original"
      disabled={undefined}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      type="checkbox"
    />
  </span>
  <span
    className="el-checkbox__label"
  >
    Checkbox
  </span>
</label>
`;
